src/kOS/Function/PrintList.cs

```diff
@@ -276,7 +273,7 @@ private kList GetConfigList()
             list.AddColumn("Name", 34, ColumnAlignment.Left);
             list.AddColumn("Value", 6, ColumnAlignment.Left);
 
-            foreach (ConfigKey key in Config.Instance.GetConfigKeys())
+            foreach (ConfigKey key in SafeHouse.Config.GetConfigKeys())
```

Comment:
Ok, I've put it back. Shouldn't have copied over the entire file when the difference on my side was a forgotten 'uses' reference.